### PR TITLE
Builtin: Italian translation

### PR DESCRIPTION
**Pull request by Zughy**
## To do

This PR is Ready for Review.

## How to test
I mean... you should know Italian. I've translated and a friend of mine checked it, so it should be good

### CODE REVIEW

File: builtin/locale/__builtin.it.tr [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+Command=Comando
+Parameters=Parametri
+For more information, click on any entry in the list.=Per più informazioni, clicca su una qualsiasi voce dell'elenco.
+Double-click to copy the entry to the chat history.=Doppio click per copiare la voce nella cronologia della chat.
+Command: @1 @2=Comando: @1 @2
+Available commands: (see also: /help <cmd>)=Comandi disponibili: (vedi anche /help <comando>)
+Close=Chiudi
+Privilege=Privilegio
+Description=Descrizione
+print [<filter>] | dump [<filter>] | save [<format> [<filter>]] | reset=print [<filtro>] | dump [<filtro>] | save [<formato> [<filtro>]] | reset=
```
Comment (SmallJoker) [inline builtin/locale/__builtin.it.tr:196]: Tailing =

Comment (rubenwardy) [reply builtin/locale/__builtin.it.tr:196]: This is fine (and good) for untranslated things

Comment (SmallJoker) [reply builtin/locale/__builtin.it.tr:196]: except that it *is* translated

Comment (Zughy) [reply builtin/locale/__builtin.it.tr]: whoops, thanks, fixed

File: builtin/locale/__builtin.it.tr [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+You don't have permission to teleport other players (missing privilege: @1).=Non hai il permesso di teletrasportare altri giocatori (privilegio mancante: @1).
+([-n] <name> <value>) | <name>=([-n] <nome> <valore>) | <nome>
+Set or read server configuration setting=Imposta o ottieni le configurazioni del server
+Failed. Use '/set -n <name> <value>' to create a new setting.=Errore. Usa 'set -n <nome> <valore>' per creare una nuova impostazione
+@1 @= @2=@1 @= @2
+<not set>=<non impostato>
+Invalid parameters (see /help set).=Parametri non validi (vedi /help set).
+Finished emerging @1 blocks in @2ms.=Finito di emergere @1 blocchi in @2ms
+emergeblocks update: @1/@2 blocks emerged (@3%)=aggiornamento emergeblocks: @1/@2 blocchi emersi (@3%)
+(here [<radius>]) | (<pos1> <pos2>)=(qui [<raggio>]) | (<pos1> <pos2>)
```
Comment (sfan5) [inline builtin/locale/__builtin.it.tr:72]: `here` is part of the syntax and must not be translated

File: builtin/locale/__builtin.it.tr [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+Show or set time of day=Mostra o imposta l'orario della giornata
+Current time is @1:@2.=Orario corrente: @1:@2.
+You don't have permission to run this command (missing privilege: @1).=Non hai il permesso di eseguire questo comando (privilegio mancante: @1) 
+Invalid time.=Orario non valido.
+Time of day changed.=Orario della giornata cambiato.
+Invalid hour (must be between 0 and 23 inclusive).=Ora non valida (deve essere tra 0 e 23 inclusi)
+Invalid minute (must be between 0 and 59 inclusive).=Minuto non valido (deve essere tra 0 e 59 inclusi)
+Show day count since world creation=Mostra il conteggio dei giorni da quando il mondo è stato creato
+Current day is @1.=Giorno attuale: @1.
+[<delay_in_seconds> | -1] [reconnect] [<message>]=[<ritardo_in_secondi> | -1] [riconnetti] [<messaggio>]
```
Comment (sfan5) [inline builtin/locale/__builtin.it.tr:138]: `reconnect` here too

**Comment by sfan5** [comment]
You committed a `fonts/Unifont.ttf`, please remove that too.

**Comment by Zughy** [comment]
all issues addressed

**Review by sfan5 — APPROVED** [review]
Can't review the translation but looks correct formally.

**Review by SmallJoker — APPROVED** [review]
LGTM

**Pull request merged**